src/Routes.js

```diff
@@ -23,9 +17,9 @@ export const Routes = () => (
                     {...path}
                 />
             ))}
-            <Redirect path="/signatures" to={`${paths[1].path}`} push />
+            <Redirect path="/signatures" to={`${paths[0].path}`} push />
             {/* Finally, catch all unmatched routes */}
-            <Redirect path="*" to={`${paths[1].path}`} push />
+            <Redirect path="*" to={`${paths[0].path}`} push />
```

Comment:
This is more of a nitpick, but if paths const is just going to be 1 object, why keep it as an array? Unless we plan to change this again later

Reply:
This code is one-off and will never be modified, when malware is GA, this temporary page will be replaced by the actual Malware app (which is in prod/beta)